packages/selector-ui/src/components/wallet-selector/ledger-derivation-path/ledger-derivation-path.tsx

```diff
@@ -0,0 +1,90 @@
+import { FunctionalComponent, h } from "@stencil/core";
+import appState from "../../../store";
+
+interface LedgerDerivationPathProps {
+  onBack: () => void;
+  onConnected: () => void;
+}
+
+export const LedgerDerivationPath: FunctionalComponent<
+  LedgerDerivationPathProps
+> = ({ onBack, onConnected }) => {
+  const resetState = () => {
+    appState.ledgerIsLoading = false;
+    appState.ledgerError = "";
+  };
+
+  const handleDerivationPathChange = (e) => {
+    appState.ledgerDerivationPaths = [e.target.value];
+  };
+
+  const handleConnectClick = async () => {
+    appState.ledgerIsLoading = true;
+    // TODO: Can't assume "ledger" once we implement more hardware wallets.
+    const wallet = await appState.selector.wallet("ledger");
+
+    if (wallet.type !== "hardware") {
+      return;
+    }
+
+    appState.ledgerIsLoading = true;
+
+    return wallet
+      .connect({ derivationPaths: [appState.ledgerDerivationPaths][0] })
```

Comment:
I think this a typo?  It should be `derivationPaths: [appState.ledgerDerivationPaths[0]]` or simply `derivationPaths: appState.ledgerDerivationPaths`

Reply:
Yes it's a typo I will fix it. For now I am just adding the first element of derivationPaths but I see I did it wrongly.